src/pm/hydra/proxy/pmip_cb.c

```diff
@@ -331,7 +331,7 @@ static HYD_status pmi_cb(int fd, HYD_event_t events, void *userp)
     int repeat;
     do {
         int buflen = 0;
-        int pmi_version;
+        int pmi_version = 0;
```

Comment:
Don't init here since it is not clear why `check_pmi_cmd` will miss it. Do it in `check_pmi_cmd` instead.

Reply:
Done.